style.css

```diff
@@ -1,6 +1,6 @@
 :root {
   --em-size: 16px;
-  --font-stack: system-ui, sans-serif;
+  --font-stack: -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
```

Comment:
This font stack is missing my system font (Cantarell, the default Gnome font on Fedora, Arch, or any other Linux using pure Gnome, unlike Ubuntu). Could you add it just before Ubuntu?

`-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Cantarell, Ubuntu, roboto, noto, arial, sans-serif;`

Reply:
Added!